### PR TITLE
Add first blog post for team clojurians

### PR DESCRIPTION
**Pull request by saskali**

### CODE REVIEW

File: blog/_posts/2017-07-06-introducing-team-clojurians.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: Introducing Team Clojurians
+layout: post
+created_at: Thu July 6 2017
+permalink: blog/introduction
```
Comment (alicetragedy) [inline blog/_posts/2017-07-06-introducing-team-clojurians.md]: could you change this permalink to `blog/2017-07-06-introducing-team-clojurians`? thank you!

File: blog/_posts/2017-07-06-introducing-team-clojurians.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+**Saskia**: fueled by movement, connection and music - huge curiosity for life - exploring learning sharing - will finish my studies this summer!
+
+## so how did we meet??
+
+We have [ClojureBridge Berlin](http://clojurebridge-berlin.org/) and the greater [Clojure Berlin community](https://www.meetup.com/Clojure-Berlin) along with Twitter dot com to thank for connecting us!
+
+Sometime last year, we'd both seen _the same tweet_ about RGSoC from our mutual friend and notable clojurian [Arne](https://twitter.com/plexus):
+
+![plexus rgsoc tweet](/img/blog/2016/clj-plexus-rgsoc-tweet.jpg)
```
Comment (alicetragedy) [inline blog/_posts/2017-07-06-introducing-team-clojurians.md]: the url for the image has been mistyped, it should be `/img/blog/2017/..`

File: blog/_posts/2017-07-06-introducing-team-clojurians.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+We have [ClojureBridge Berlin](http://clojurebridge-berlin.org/) and the greater [Clojure Berlin community](https://www.meetup.com/Clojure-Berlin) along with Twitter dot com to thank for connecting us!
+
+Sometime last year, we'd both seen _the same tweet_ about RGSoC from our mutual friend and notable clojurian [Arne](https://twitter.com/plexus):
+
+![plexus rgsoc tweet](/img/blog/2016/clj-plexus-rgsoc-tweet.jpg) 
+*screenshot taken by Chris Sun*  
+
+**Saskia**: RGSoC and re-frame had been on the back of my head for a while already, as I remember having seen that tweet. I thought I wouldn't be able to participate, as I had a job that I wanted to continue over the summer. To my surprise and luck, my job ended one day before the applications opened. From there, everything seemed to fall into place. We didn't know each other, but a friend pointed me (Saskia) to this tweet that Chris had posted:
+
+![looking for teammate tweet](/img/blog/2016/clj-teammate-tweet.png)
```
Comment (alicetragedy) [inline blog/_posts/2017-07-06-introducing-team-clojurians.md]: the url for the image has been mistyped, it should be `/img/blog/2017/..`

File: blog/_posts/2017-07-06-introducing-team-clojurians.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+The first time we met in person was last month! We immediately got along super well and went climbing at the boulderhalle (because of course), ate tasty vegetarian food, and talked about life. We'll complement each other well because Saskia has more experience with clojure and Chris has experience with browsers and javascript, and we'll meet in the middle with clojurescript!
+
+
+## companies, coaches, mentors, & supervisor
+
+We spend our work week at two different companies in the heart of Berlin nearby many of our favorite places. We feel very lucky to have the opportunity to work in such a great and beautiful environment where we can get some fresh air walking along the 'Landwehrkanal' or sitting on the balcony overlooking the roofs of the city.
+
+Most days we're at [NextJournal](https://nextjournal.com), which actually uses Clojurescript and re-frame for their front-end, so we have opportunities to exchange ideas and ask upcoming questions! We also visit [bitcrowd](https://bitcrowd.net/en), which frequently also hosts Clojure and Ruby community events in their office with a beautiful rooftop balcony.
+
+![courtyard](/img/blog/2016/clj-sky.jpg)
```
Comment (alicetragedy) [inline blog/_posts/2017-07-06-introducing-team-clojurians.md]: the url for the image has been mistyped, it should be `/img/blog/2017/..`

**Review by alicetragedy — CHANGES_REQUESTED** [review]
Awesome work! thanks for the great post @sklind. I've left a couple of comments regarding things that need changing, once that's done and the new changes are pushed I think we can merge and share this post. please let me know if you have any questions!

**Review by alicetragedy — APPROVED** [review]
great work! we'll merge and share this tomorrow. Thanks for making the changes to your PR so quickly!

**Comment by saskali** [comment]
yay thanks for reviewing! :smiley: :100:

**Pull request merged**